guard-lambda/src/main.rs

```diff
@@ -47,7 +93,38 @@ pub(crate) async fn call_cfn_guard(e: CustomEvent, _c: Context) -> Result<Custom
         let json_value: serde_json::Value = serde_json::from_str(&result)?;
         results_vec.push(json_value)
     }
+
+    //let response: CustomOutput;
+    let mut resp = Vec::new();
+
+    if e.s3_output_bucket.is_empty() {
+        resp = results_vec;
+    }else {
+        // No extra configuration is needed as long as your Lambda has 
```

Comment:
If there are prerequisite permissions to be granted to Lambda role, let's add a documentation in README for Lambda for granting required permissions to the access policy such as bucket `s3:PutObject` action.